definitions/tools/duphold.cwl

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: "/usr/bin/duphold"
+arguments: ["--threads", "$(runtime.cores)"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 10000
+    - class: DockerRequirement
+      dockerPull: "mgibio/duphold-cwl:0.1.4"
+
+inputs:
+    bam:
+        type: File
+        inputBinding:
+            position: 1
+            prefix: "--bam"
+        doc: "aligned bam/cram file"
+    output_vcf_name:
+        type: string?
+        default: "duphold_annotated.vcf"
+        inputBinding:
+            position: 4
```

Comment:
It's harmless, but just noting that two of the inputs have position `4`.

Reply:
Good catch. I moved the inputs around to get them in alphabetical order and didn't check/update the positions